src/web/constants.ts

```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line import/prefer-default-export
+export const FIBONACCI = 'FIBONACCI';
+export const T_SHIRT = 'TSHIRT';
```

Comment:
We have Fibonacci and TShirt in the `VoteSequenceType` enum, what do we need them here for?

Reply:
Good point. I'll remove them.